apps/asap-server/test/controllers/teams.test.ts

```diff
@@ -303,51 +303,90 @@ describe('Team controller', () => {
       });
     });
 
-    test('Should parse the team user correctly when the avatar is null', async () => {
-      const teamId = 'team-id-1';
-      const user = fetchByIdUserResponse.items[0];
-      user.data.avatar.iv = null as any;
-
-      const userResponse = {
-        total: 1,
-        items: [user],
+    describe('Avatar', () => {
+      const user = {
+        ...fetchByIdUserResponse.items[0],
+        data: {
+          ...fetchByIdUserResponse.items[0].data,
+          avatar: { iv: 'test avatar' },
+        },
       };
 
-      nock(config.baseUrl)
-        .post(`/api/content/${config.appName}/graphql`, {
-          query: buildGraphQLQueryFetchTeam(teamId),
-        })
-        .reply(200, graphQlTeamResponse)
-        .get(`/api/content/${config.appName}/users`)
-        .query({
-          $filter: `data/teams/iv/id eq '${teamId}'`,
-        })
-        .reply(200, userResponse);
+      test('Should parse the team user correctly when the avatar is null', async () => {
+        const teamId = 'team-id-1';
+
+        user.data.avatar.iv = null as any;
```

Comment:
Since we need to mutate the base object, we maybe need a deep copy, or these tests could use different objects.
Won't the tests fail if you run the `delete` one before the assign no `null`?

Reply:
yes, hence the copy:
https://github.com/yldio/asap-hub/pull/840/files/238050457b37380277be8eb4524699bd66498447#diff-79015ebee0b55ad2336fccbd627b61384c730904ec40c7cf86a3ad57c904d90eR307

Reply:
I see, but my point was not file-wide, was only for the `avatar` tests. 
I think this is fine for now